scripts/set-backend-env-vars.sh

```diff
@@ -19,6 +20,16 @@ else
   FRONTEND_BASE_URL="$DEFAULT_FRONTEND_ROUTE"
 fi
 
+# Dynamically set DJANGO_CONFIGURATION based on Cloud.gov Space
```

Comment:
This sets us up directly for https://github.com/raft-tech/tanf-app/issues/970 along with utilizing the new settings module for Cloud.gov deployed environments.